packages/IMAPClient-Core.package/ICMovedEmailStorage.class/instance/getSetForAccount.andFolder..st

```diff
@@ -0,0 +1,6 @@
+as yet unclassified
+getSetForAccount: anAccountName andFolder: aFolder
```

Comment:
`get` prefix in method name is "unsqueakish" (apparently)

also shares lots of code with `addMail:forAccount:` (should probably be used in that method)

Reply:
anyone any suggestions for a better name? :D

Reply:
`movedMailsForAccount:andFolder:` maybe?

Reply:
good idea, done